presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoInformer.java

```diff
@@ -52,21 +51,17 @@ public int getPrestoPort()
 
     /**
      * Returns whether Presto is running as a coordinator
-     *
+     *<p>
+     * Always check whether Presto is running using {@link PackageController#isRunning()}
```

Comment:
You don't need to return false, but if a method is making assumptions, it's good to put in a `checkState()` that those assumptions hold.  You can catch the exception from isRunning() so that you don't need to rethrow it, and fail the checkState if you either encounter an exception or `isRunning()` returns false.